### DISCUSSION TITLE
微米符号不能正常显示

### DISCUSSION THREAD

**Discussion by MichaelJJ008** [sjtug/SJTUThesis#804] [提问] [answered]
如下图，希腊字母μ和Ω无法显示，把\micro换成\mu会变成斜体μ，不符合单位规范
![c440993696833043c700cff25b81fbf](https://user-images.githubusercontent.com/71876253/167866862-a06cdb8d-7e27-4312-bee1-b37f7bdb6148.png)
![b1b65f7343453cfaa6a10b161e653cf](https://user-images.githubusercontent.com/71876253/167866876-3f687f2d-2e07-4979-b208-d648392d61f8.png)
使用的是texlive2019+texstudio

**Comment by LogCreative** [comment, marked as answer]
尝试更新 TeX Live 至最新版本（2022），~以使得 `siunix` 版本大于 3.0。~

https://github.com/sjtug/SJTUThesis/blob/b164d0f5b3087ed86c1038b5a85014b47a92c6f2/contents/math_and_citations.tex#L17

**Reply by MichaelJJ008** [reply]
好的，我卸载重装新版本试试

**Reply by MichaelJJ008** [reply]
版本升级后已解决，谢谢

**Comment by AlexaraWu** [comment]
siunix 2.7 和之前的版本 `\micro` 和 `\Ohm` 都有点问题，2.8 版 (TeXLive 2020 及之后) 基本修复了。和 3.0 什么的关系不大。因为 3.0 改了大量接口，为了兼容模板目前其实用的还是 v2。
```latex
\usepackage{siunitx}[=v2]
```

还在用 TeXLive 2019 的可以用 @stone-zeng 在 https://github.com/sjtug/SJTUThesis/issues/439#issuecomment-511723467 里提供的代码修补一下。

**Reply by MichaelJJ008** [reply]
谢谢，我升级版本后问题解决了

**Comment by MichaelJJ008** [comment]
感谢大佬们的解答，很有帮助
感谢模板的创建和维护者们，非常好用
